### DISCUSSION TITLE
Vault with Edge Functions

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#18876] [Questions]
Hi, Can the Vault be accessed by Edge functions?  I would like to store 3rd Party API keys that a called by Edge functions from a webhook, or is there another/better way of doing this?

**Comment by therealpurplemana** [comment]
Yes

**Comment by naimkhrof** [comment]
> Hi, Can the Vault be accessed by Edge functions? I would like to store 3rd Party API keys that a called by Edge functions from a webhook, or is there another/better way of doing this?

Hello, did you succeed ?

**Comment by tomaspozo** [comment]
Better to use [Edge Function Secrets](https://supabase.com/docs/guides/functions/secrets). 

Vault is for secrets in the Database space, as the docs indicate:
![image](https://github.com/user-attachments/assets/6b71688b-f40f-4b9d-b0f6-72914b4f77ec)
